applications/GeoMechanicsApplication/tests/test_dynamics.py

```diff
@@ -100,12 +100,6 @@ def test_load_on_block_2d_no_damping(self):
         for node in nodes:
             self.assertVectorAlmostEqual(calculated_result[node][what], expected_result[node][what])
 
-    @KratosUnittest.skip("unit test skipped as it is not ready")
```

Comment:
Do we see that there is a need for an undrained linear elastic soil? If so, I'd propose to create a draft issue for it. I agree with removing this now, should we also remove the test_1d_confined_undrained_wave.gid folder, or is that not even there yet?

Reply:
The undrained folder is not present, so I removed the skipped reference to it.